nixos/modules/virtualisation/qemu-vm.nix

```diff
@@ -142,7 +145,7 @@ let
           buildInputs = [ pkgs.utillinux ];
           QEMU_OPTS = if cfg.useEFIBoot
                       then "-pflash $out/bios.bin -nographic -serial pty"
-                      else "-nographic -serial pty";
+                      else "-nographic -serial mon:stdio";
```

Comment:
Why is only this side of the branch needing `mon:stdio`?

And what's up with `pty`, does it fail?

Reply:
`pty` silently fails, making debug much more difficult
and yeah, i should probably fix the other branch also